Druid/DRBytecodeIRGenerator.class.st

```diff
@@ -308,21 +308,18 @@ DRBytecodeIRGenerator >> interpretPushWith: aRBMessageNode [
 DRBytecodeIRGenerator >> interpretRememberWith: aRBMessageNode [
 
 	| obj |
-	obj := (self visitOperand: aRBMessageNode arguments first)
-		       simpleConstantFold.
-
+	obj := self visitOperand: aRBMessageNode arguments first.
+	"	
+	cogit 	backEnd saveAndRestoreLinkRegAround: [cogit CallRT: ceStoreCheckTrampoline].
```

Comment:
comment?

Reply:
The comment is the code to generate (so why `saveLinkReg`)